tools/kokoro/unit-test/test.sh

```diff
@@ -1,16 +1,24 @@
-# Copyright 2018 Google Inc. All Rights Reserved.
+#!/bin/bash
+
```

Comment:
nit: empty after #! here but not in trampoline.sh. can you please make it consistent?

Reply:
Done. Tried to consistently add line after #!/bin/bash.